index.js

```diff
@@ -35,6 +35,7 @@ function Account (options) {
     on: events.on.bind(this, state),
     one: events.one.bind(this, state),
     off: events.off.bind(this, state),
-    validate: require('./lib/validate').bind(this, state)
+    validate: require('./lib/validate').bind(this, state),
+    update: require('./lib/update').bind(this, state)
```

Comment:
sorry for the nitpicking, but could you please move `update` below `fetch` here as well?